whois-update/src/main/java/net/ripe/db/whois/update/keycert/PgpPublicKeyWrapper.java

```diff
@@ -112,9 +117,10 @@ public static PgpPublicKeyWrapper parse(final RpslObject object) {
             }
 
             return new PgpPublicKeyWrapper(masterKey, subKeys);
-        } catch (IOException e) {
+        } catch (IOException | PGPException e) {
```

Comment:
IOException and PGPException are already known side-effects, which don't need to be logged (i.e. the keyring is invalid). We only need to log for other Exceptions, this is why it's caught separately.

Reply:
This situation happened once in production, due to a malformed key-cert object with a quoted-printable encoded character at the beginning of the keycert, there's a test case which reproduces it. For some reason, Bouncy Castle returns "null" for an unknown packet, rather than throwing an IOException or PGPException.